pkg/controller/certificaterequests/sync_test.go

```diff
@@ -276,7 +276,14 @@ func TestSync(t *testing.T) {
 				},
 			},
 			Builder: &testpkg.Builder{
-				CertManagerObjects: []runtime.Object{gen.CertificateRequest("test")},
+				CertManagerObjects: []runtime.Object{gen.CertificateRequest("test"),
+					gen.Issuer("ca-issuer",
+						gen.AddIssuerCondition(cmapi.IssuerCondition{
+							Type:   cmapi.IssuerConditionReady,
+							Status: cmapi.ConditionTrue,
+						}),
+						gen.SetIssuerCA(cmapi.CAIssuer{}),
```

Comment:
It looks like the issuer has changed from being a selfsigned issuer to a CA issuer. Also, this CAIssuer doesn't specify a SecretName. Is that intentional?